src/jog/jog.service.ts

```diff
@@ -0,0 +1,56 @@
+import {BadRequestException, Injectable, NotFoundException} from '@nestjs/common';
+import {InjectModel} from "@nestjs/mongoose";
+import * as mongoose from "mongoose";
+import {Jog} from "./schemas/jog.schema";
+import {Query} from 'express-serve-static-core';
+
+
+@Injectable()
+export class JogService {
+    constructor(
+        @InjectModel(Jog.name)
+        private jogModel: mongoose.Model<Jog>
+    ) {}
+
+    async findAll(query: Query): Promise<Jog[]> {
+
+        const resultsPerPage = 2;
```

Comment:
How did we decide to have only 2 per page?

Reply:
Randomly, to be frank. I searched for some sort of calculation algorithm but I'm not very sure its needed here? Is there a known convention for pagination that i didnt come across in my search? Most of the formulas online to have a set page size.